### ISSUE TITLE
Issue when removing languages

### ISSUE THREAD

**Issue by sandiwinter** [polylang/polylang#433]
## Prerequisites

- [x] This is not a usage question (Those should be directed to the [community supported forum](https://wordpress.org/support/plugin/polylang), unless this is a question about Polylang Pro in which case you should [use the helpdesk](https://polylang.pro/support/)).
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest `master` branch of Polylang and the latest WordPress version.
- [x] This bug happens with only Polylang plugin active
- [x] This bug happens with a default WordPress theme active
- [x] I can reproduce this bug consistently

## Steps to reproduce the issue

1. Add few langs
2. Associate few widgets to this langs
3. Remove langs, leave only one

## Expected behavior and actual behavior

When I follow those steps, I see...

Widget stay and causing multiple widget on primary language, on website and in dash

I was expecting...

Remove widgets associated to removed langs

**Comment by Chouby** [comment]
This is currently the expected behavior. When we remove a language, the associations with this language are removed, but the content itself is never removed. Posts, terms, menus and widgets are all kept, without language. 

It's true however that, in the case of the posts, terms and menus, the content without language is not displayed on frontend, while widgets without language are displayed in all languages. 

I believe however that it would not be a good thing to oblige the users to assign a language to all widgets to display them on the frontend (and thus not to display widgets without language). And I 'm also not comfortable with content removal.

**Comment by sandiwinter** [comment]
Widgets without language should not be visible on frontend, because in such case we getting infinite scrolling website with hundred same/repeating widgets.

**Comment by JanWebofka** [comment]
Hi guys,

i have same problem.. we get duplicated products after deleting language, u cant delete them in bulk if sync is on them 

![Sync-inproduct](https://user-images.githubusercontent.com/101801230/206867884-234f1449-8fad-4b8f-94d9-06473e13211a.png)

We need -> bult turn of sync between posts and delete all content assigned to language

@polylang

**Comment by drawesome** [comment]
yes! please. i have a site that EVERYTHING is 'synced' (for some reason the client wanted UK english and US english).

Hundreds of Posts, Custom Post Types, Pages, Taxonomies, Events - all with synced translations. 

Now they want it gone, but trying to bulk delete anything that has been synced causes EVERYTHING to be deleted.   

Turning off syncing just results in EVERYTHING being duplicated - not great.

How are we supposed to delete things when there is no simple way to disable post syncing while also unpublishing content.
